config/locales/views.fr.yml

```
@@ -428,6 +428,8 @@ fr:
     show:
       title: Relances / %{full_name}
   solicitations:
+    create:
+      thanks: Merci ! On vous rappelle rapidement pour le prédiagnostic. L’équipe Place des Entreprises
```

Il me semble que @Adeline-Lrn voulait réécrire cette phrase? (et il manque un point final 👅 )
```suggestion
      thanks: Merci ! On vous rappelle rapidement pour le prédiagnostic. L’équipe Place des Entreprises.
```